src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -1698,6 +1711,78 @@ private int GetTheBestCandidateIndex<TMember>(ArrayBuilder<MemberResolutionResul
             return currentBestIndex;
         }
 
+        private static void RemoveLowerPriorityMembers<TMember>(ArrayBuilder<MemberResolutionResult<TMember>> results)
```

Comment:
This method, and the calls to this method in the current file, are the actually interesting parts of the feature. This implements the rules from the speclet, and I've tried to be perf conscious with a couple of early outs to avoid any allocations in the very common scenarios where we have no adjusted priorities.